app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -72,10 +85,11 @@ examples:
         - label: "Blue"
           value: "blue"
   without_hint_text:
-    description: Hint text defaults to 'Select all that apply'. This can be removed entirely with this option.
+    description: Hint text defaults to 'Select all that apply'. This can be removed entirely with this option. Note that this option can be combined with the visually_hide_heading option, as shown.
```

Comment:
The first sentence (`Hint text defaults to 'Select all that apply'.`) initially confused me - I thought it meant that in this example, the hint text defaulted to "Select all that apply". We could move this to the `with_custom_hint_text` example instead.